src/pages/bookmark/Bookmark.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react'
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Posts, Sidebaar } from '../../component';
+
+export const Bookmark = () => {
+    const {bookmarks} = useSelector((state) => state.bookmark);
+    const [ bookMarkPosts, setBookMarkPosts ] = useState(bookmarks)
```

Comment:
can remove this `useState` here since it is not being used in the file

Reply:
ok I will remove it